src/modules/RA/Campaign/components/CloneCampaignButton.js

```diff
@@ -31,11 +32,18 @@ const CloneCampaignButton = ({ record, ...rest }) => {
         setViewpoints(vp);
         setLoading(false);
       } catch (err) {
+        /* Error raised from unmounted component is skipped
+          to avoid setState error on unmounted component */
+        if (!isMounted) {
```

Comment:
Should also be present before the other state changes.

Reply:
Done.

Reply:
Done.